roles/oVirt.datacenter-cleanup/tasks/remove_hosts_not_hosting_engine_vm.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Find facts for the engine VM
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }} and name={{ hosted_engine_vm_name }}"
+    fetch_nested: true
+
+- name: Show vm data
```

Comment:
Please remove,unless it's really needed.

Reply:
fixed